app/controllers/maintenance_tasks/application_controller.rb

```diff
@@ -11,7 +11,7 @@ class ApplicationController < MaintenanceTasks.parent_controller.constantize
       policy.style_src(
         BULMA_CDN,
         # ruby syntax highlighting
-        "'sha256-2AM66zjeDBmWDHyVQs45fGjYfGmjoZBwkyy5tNwIWG0='",
+        "'sha256-WHHDQLdkleXnAN5zs0GDXC5ls41CHUaVsJtVpaNx+EM='",
```

Comment:
Was this intentional? Let's add an explanation to why we are changing this

Reply:
This is changing because we added a new CSS rule to `app/views/layouts/maintenance_tasks/application.html.erb`